storybook/stories/playground/with-undo-redo/index.js

```diff
@@ -60,7 +59,6 @@ export default function EditorWithUndoRedo() {
 						label="Redo"
 					/>
 					<BlockToolbar hideDragHandle />
```

Comment:
I noticed that hideDragHandle is always passed to this component. It's only left as false in the case of the "popover toolbar" which is rendered by the framework, this means that we should probably change the default value of this prop to "true".

Reply:
Agreed.